cajita/src/Cajita.hpp

```diff
@@ -26,7 +26,11 @@
 #include <Cajita_LocalMesh.hpp>
 #include <Cajita_ManualPartitioner.hpp>
 #include <Cajita_MpiTraits.hpp>
+
+#ifdef KOKKOS_ENABLE_OPENMPTARGET // FIXME_OPENMPTARGET
```

Comment:
Based on the logic in `Cajita/src/CMakeLists.txt` don't you want to *not* include this if OpenMP Target is on?

Reply:
My bad

Reply:
Fixed